aws/resource_aws_cognito_resource_server.go

```diff
@@ -146,7 +146,9 @@ func resourceAwsCognitoResourceServerRead(d *schema.ResourceData, meta interface
 		scopeIdentifier := fmt.Sprintf("%s/%s", aws.StringValue(resp.ResourceServer.Identifier), elem["scope_name"].(string))
 		scopeIdentifiers = append(scopeIdentifiers, scopeIdentifier)
 	}
-	d.Set("scope_identifiers", scopeIdentifiers)
+	if err := d.Set("scope_identifiers", scopeIdentifiers); err != nil {
```

Comment:
Sorry this change was unrelated to the syntax fixes to try and see if the new Terraform 0.12 error could be teased out with `d.Set()`, but it didn't help. The error check is a recommended practice, but let me know if you want this removed since its unrelated.